atlp-rwanda/phoenix-bn-frontend
# Test with Jest,Git-workflow CI and Heroku

#### What does this PR do?
> This Pr configure test environment and deploys to Heroku.
>
#### Description of Task to be completed
> - Configured Testing environment with Jest & Enzyme to ensure that all functionalities are tested before deployment.
> - GitHub Actions Workflow is configured as a continuous integration service used to build and test the app.
> - Configured Heroku Pipeline on Heroku
> - Git-workflow badge has been added to your repo
> - Configure CodeClimate for code analysis and Maintainability checking.
#### How should this be manually tested?
> - clone the Branch 
> - run `npm i`
> - run `npm run test` to check tests
> - run `npm run test:coverage` to check the coverage
> - change something and push to check the Git-workflow tests
#### Any background context you want to provide?

#### What are the relevant Trello story?
> - Chore for Setup Continuous Integration & Quality Assurance pipeline use this trello link  https://trello.com/c/ZGT77jzX
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/101324563-1d5d9980-3873-11eb-91d5-c088f4622cc2.png)

>
> Thank you !!!
